docs/DeveloperGuide.md

```diff
@@ -308,7 +308,99 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case 6: View a student's details**
+
+Precondition: The student's details have been added to TutorAid.
+
+**MSS**
+
+1. Tutor requests a list of students
+
+2. TutorAid shows the list of students
+
+3. Tutor finds the specific student in the list and requests to view the student's details
+
+4. TutorAid shows the student's details
+
+   Use case ends.
+
+**Extensions**
+
+* 3a. The given index is invalid.
+
+  * 3a1. TutorAid shows an error message.
+
+    Use case resumes at step 3.
+
+**Use case 13: Set up a class**
+
+Preconditions: The students of the class have been added to TutorAid.
+
+**MSS**
+
+1. Tutor requests to add a class
+
+2. TutorAid requests for details of the class including day of the week, time and price.
+
+3. Tutor enters the details of the class.
+
+4. TutorAid creates the class.
+
+5. Tutor <u>adds a student to the class (UC14)</u>. 
+
+   Step 5 is repeated until all students have been added.
+
+   Use case ends.
+
+**Extensions**
```

Comment:
Can I check if extensions for invalid day, timings and prices need to be included as well?

Reply:
Fixed, thanks